include/Util/Color.hpp

```diff
@@ -49,12 +60,67 @@ class Color : public glm::vec4 {
      */
     SDL_Color ToSdlColor() const {
         return SDL_Color{
-            static_cast<Uint8>(r * 255),
-            static_cast<Uint8>(g * 255),
-            static_cast<Uint8>(b * 255),
-            static_cast<Uint8>(a * 255),
+            static_cast<Uint8>(r),
+            static_cast<Uint8>(g),
+            static_cast<Uint8>(b),
+            static_cast<Uint8>(a),
         };
     }
+
```

Comment:
> Detect dangling references in value handles like std::string_view. These dangling references can be a result of constructing handles from temporary values, where the temporary is destroyed soon after the handle is created.

Returning a `string_view` of a temporary value would cause undefined behavior. Would you prefer to use a `m_String` to store and return its `string_view` or just simply return a` string`?

Reply:
I prefer to use second way, btw.